packages/snap-preact-components/src/components/Molecules/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,117 @@
+/** @jsx jsx */
+import { h, Fragment } from 'preact';
+import { useState, useRef } from 'preact/hooks';
+
+import { jsx, css } from '@emotion/react';
+import classnames from 'classnames';
+import { observer } from 'mobx-react-lite';
+import deepmerge from 'deepmerge';
+
+import { Icon, IconProps } from '../../Atoms/Icon/Icon';
+import { defined } from '../../../utilities';
+import { Theme, useTheme, CacheProvider } from '../../../providers';
+import { ComponentProps, BreakpointsProps } from '../../../types';
+import { useDisplaySettings } from '../../../hooks/useDisplaySettings';
+
+const CSS = {
+	searchInput: ({ theme, subProps }) =>
+		css({
+			margin: '16px 0 0 0',
```

Comment:
This should probably not be here - but instead as a style that the facet component adds?